### ISSUE TITLE
Binary Option with BuyV2 is not working?

### ISSUE THREAD

**Issue by robotfitowner** [Lu-Yi-Hsun/iqoptionapi#164]
just i try to BuyV2 Name and same as all of paramters. but is it still working?

**Comment by Lu-Yi-Hsun** [comment]
try private version have more support

**Comment by Lu-Yi-Hsun** [comment]
@robotfitowner  you can pull request to fix this

**Comment by robotfitowner** [comment]
how can i try to  private version?

**Comment by Nova3x** [comment]
Yes, I found same problem. Now BuyV2 can't Buy Order and nothing respond back from IQOption after 04-Dec-19 evening. It was working fine before 4-Dec-19 evening. So Is any new parameter required OR changed from IQOption? Could you please suggest how to check and fix this.

**Comment by robotfitowner** [comment]
I sent c# with socket. But that’s not work
Actually I also developer so I already check response with results nothing..

2019년 12월 6일 (금) 오후 6:57, Nova3x <notifications@github.com>님이 작성:

**Comment by MightyMaxSaviorOfTheUniverse** [comment]
Hello, 

Although I'm not using **this api** to trade I have been experiencing similar problems since the second of December.

I thought I would share my findings so that together we can create a solution.

https://github.com/harwee/IQOption-Api/issues/60#issuecomment-562819584

**Comment by robotfitowner** [comment]
Yes I also like that. Because I already change of your resources. That case
api is mostly changed buy function and results.

Do you have line or any sns? We can discuss about the api problem. And also
I have chrome bot too

If you can connect me, let me know please

Thank you

2019년 12월 7일 (토) 오후 3:28, MightyMaxSaviorOfTheUniverse <
notifications@github.com>님이 작성:

**Comment by Lu-Yi-Hsun** [comment]
@robotfitowner 

I think that is working in private version .

**Comment by robotfitowner** [comment]
Yes working but buyresult function is not working
Already new message they are response

2019년 12월 7일 (토) 오후 6:16, Anson <notifications@github.com>님이 작성:

**Comment by Lu-Yi-Hsun** [comment]
@robotfitowner 

What api not work? 
Check win?

I can fix any api you want in private version

**Comment by Lu-Yi-Hsun** [comment]
@MightyMaxSaviorOfTheUniverse 
How about try private version

**Comment by robotfitowner** [comment]
Really good but I hope know how to copy trading with worldwide top users.
Just write user id only?



2019년 12월 8일 (일) 오후 5:26, Anson <notifications@github.com>님이 작성:

**Comment by Lu-Yi-Hsun** [comment]
@robotfitowner that is copy trade with top 100.

 if you want copy one user by Id you can rewrite the source code

**Comment by robotfitowner** [comment]
Ohh! Now setting is top 100??

2019년 12월 8일 (일) 오후 6:10, Anson <notifications@github.com>님이 작성:

**Comment by MightyMaxSaviorOfTheUniverse** [comment]
> @robotfitowner is private version that is copy trade with top 100.
> 
> if you want copy one user by Id you can rewrite the source code

Ok, Send me the private version please

**Comment by robotfitowner** [comment]
Sorry about that

2019년 12월 8일 (일) 오후 11:40, MightyMaxSaviorOfTheUniverse <
notifications@github.com>님이 작성:
